modules/github/main.tf

```
@@ -54,7 +55,8 @@ resource "null_resource" "post_repo_creation" {
       git init
       git config user.email "nomail@dbt.com"
       git config user.name "Created by Terraform"
-      git add * .*
+      git add *
+      git add .
```

```suggestion
      git add -A
```

I checked online and the consensus seems to be to use `-A` or `--all` to add all files

---

Nice!  Great to know!